lms/djangoapps/verify_student/models.py

```diff
@@ -435,6 +436,12 @@ def system_error(self,
         self.status = "must_retry"
         self.save()
 
+    def retire_user(self):
```

Comment:
Looking at this again, I think it's okay to just remove this and call `delete_by_user_value` directly?

Reply:
Sure! I'm game for that. The ticket said `Create a class method in PhotoVerification called retire_user that takes a user_id as its only parameter.` so I'm just doing what I'm told here man.

Reply:
Oh, I didn't see that. I just wrote this comment because I picked up another of these tickets https://openedx.atlassian.net/browse/EDUCATOR-2731 that said to mimic what was done for a third ticket, that just uses the `DeletableByUserValue` mixin method directly. I'm good with any version that gets the job done though.